include/cupla/cudaToCupla/driverTypes.hpp

```diff
@@ -107,6 +107,14 @@
 
 #define uint3 ::cupla::uint3
 
+#if !defined(ALPAKA_ACC_GPU_CUDA_ENABLED) && !defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+struct float3 {float x,y,z;};
+float3 make_float3(float x, float y, float z){ return float3{x,y,z}; }
```

Comment:
The functions should be `inline`, since are defined in the header file.

Reply:
yep, maybe check the original CUDA headers for what they do as well?